xblockutils/studio_editable.py

```diff
@@ -0,0 +1,295 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2015 OpenCraft
```

Comment:
Other files, eg `xblockutils/base_test.py` are putting (C) edX; in edx-platform, we don't add a copyright notice at all. I'm not sure what the right thing to do here is - @antoviaque do you know?